### PR TITLE
feat: setup pilot environments

### PR DESCRIPTION
**Pull request by jdvlio**
### Overview

Configure new environments for the Bhutan and Palau pilots.

#### ~Anchors and merge types~

~For improved readability, [YAML anchors](https://yaml.org/spec/1.2.2/#692-node-anchors) are employed to facilitate the reuse of common configuration options in the Docker Compose setup.  Compose files support the [YAML merge type](https://yaml.org/type/merge.html) to allow the override of defaults set via anchors.~

#### Merging Compose Files

The `COMPOSE_FILE` environment variable is used to specify a colon-separated list of compose files.  They are merged in order of appearance, later files overriding settings that appear in earlier files.  In the context of the setup as it is at present this variable needs to be set as
```bash
COMPOSE_FILE="docker-compose.yaml:docker-compose.bhutan.yaml:docker-compose.palau.yaml"
export COMPOSE_FILE
```
It is required that this variable is set before running any `docker compose` subcommands.

This value is set in the CI using an environment secret, since GH does not support non-secret environment variables outside of the workflow files checked into the repository itself.

### CODE REVIEW

**Comment by billguo99** [comment]
https://docs.docker.com/compose/compose-file/#fragments

File: docker-compose.palau.yaml [hunk 1]
```diff
@@ -0,0 +1,147 @@
...
+      - wallet-palau-staging
+    read_only: true
+    init: true
+
+  wallet-palau-demo-api:
+    image: ghcr.io/ntls-io/sgx-wallet-hw:demo
+    environment:
+      BIND_ADDR: "0.0.0.0:8080"
+      CERT_NAME: ntls-api
+      VIRTUAL_HOST: wallet-palau-staging-api.ntls.io
```
Comment (billguo99) [inline docker-compose.palau.yaml]: should this be `wallet-palau-api.ntls.io`?

File: docker-compose.bhutan.yaml [hunk 2]
```diff
@@ -0,0 +1,147 @@
...
+      - wallet-bhutan-staging
+    read_only: true
+    init: true
+
+  wallet-bhutan-demo-api:
+    image: ghcr.io/ntls-io/sgx-wallet-hw:demo
+    environment:
+      BIND_ADDR: "0.0.0.0:8080"
+      CERT_NAME: ntls-api
+      VIRTUAL_HOST: wallet-bhutan-staging-api.ntls.io
```
Comment (billguo99) [inline docker-compose.bhutan.yaml]: should this be `wallet-bhutan-api.ntls.io`?

**Review by billguo99 — CHANGES_REQUESTED** [review]

File: docker-compose.palau.yaml (hunk 1 above)
Comment (jdvlio) [reply docker-compose.palau.yaml]: Yes, good catch!

File: docker-compose.bhutan.yaml (hunk 2 above)
Comment (jdvlio) [reply docker-compose.bhutan.yaml]: Ditto here as well

**Review by billguo99 — APPROVED** [review]

**Pull request merged**